test/suites/para/test_tanssi_containers.ts

```diff
@@ -177,5 +190,21 @@ describeSuite({
         expect(authorities.toJSON().includes(author.toString())).to.be.true;
       },
     });
+
+    it({
+      id: "T10",
+      title: "Transactions can be made with ethers",
+      timeout: 30000,
+      test: async function () {
+        const randomAccount = generateKeyringPair();
+        let tx = await createTransfer(context, randomAccount.address, 1_000_000_000_000, { gasPrice: MIN_GAS_PRICE });
+        console.log("here 2")
+        await customWeb3Request(context.web3(), "eth_sendRawTransaction", [
+          tx,
+        ]); 
+        await context.waitBlock(2, "Container2001");
```

Comment:
I guess ideally this should wait until the transaction is included in a block? I made a helper function for that in my other PR, but it requires you to use the polkadot js api (signAndSend), so not sure if it can be used here.

https://github.com/moondance-labs/tanssi/blob/313a0a03652ae065af8bfe89e2fb2aa0688877ac/test/util/block.ts#LL64C1-L64C1